pants

```diff
@@ -34,10 +34,12 @@ fi
 
 PANTS_BOOTSTRAP="${PANTS_SETUP_CACHE}/bootstrap-$(uname -s)-$(uname -m)"
 
-VENV_VERSION=${VENV_VERSION:-16.4.3}
-
-VENV_PACKAGE=virtualenv-${VENV_VERSION}
-VENV_TARBALL=${VENV_PACKAGE}.tar.gz
+# Note that this URL points to the zipapp for the latest version of virtualenv
+# that supports the Python version specified in the URL. So it can be set to
+# the lowest version of Python we support running Pants on.
+# See https://virtualenv.pypa.io/en/latest/installation.html#via-zipapp
+VIRTUALENV_ZIPAPP=virtualenv.pyz
+VIRTUALENV_ZIPAPP_URL="https://bootstrap.pypa.io/virtualenv/3.6/${VIRTUALENV_ZIPAPP}"
```

Comment:
The lowest version we currently support, IIUC, is 3.7. Is 3.6 here to support holders versions of Pants? If so can you note how old?

Reply:
Yep, see https://github.com/pantsbuild/setup/blob/gh-pages/pants#L125  which has logic to map pants versions to supported python versions. 

But note that the current virtualenv release still supports all the way back to 2.7, so we're good for a while...